.azure-devops/test.yml

```diff
@@ -0,0 +1,15 @@
+steps:
```

Comment:
Should we remove these comments if they are not going to be used?

Reply:
Meant to revert to the commented code before.  Will update for merge once I verify in the base template.